manual/python/index.md

```diff
@@ -60,9 +60,87 @@ caption="Example of graphics generated with PyROOT."
+
+- `ShutDown` (default `True`): when PyROOT is terminating, during its cleanup phase, the ROOT C++ interpreter is shut down. If PyROOT is executed as part of a longer-running application that needs the C++ interpreter, `PyConfig.Shutdown` can be set to `False` to prevent that shutdown.
+
+- `StartGUIThread` (default `True`): when not executing in [interactive mode](https://docs.python.org/3/tutorial/interpreter.html#interactive-mode), PyROOT starts a thread that periodically polls for ROOT events (e.g. GUI events) to process them. If a given PyROOT application does not need this event processing, it can prevent the creation of the thread by setting `PyConfig.StartGUIThread` to `False`.
+
+### Enabling batch mode
+
+When running in batch mode, PyROOT will not display any graphics. There are two ways in which a user can activate the batch mode:
+
+1. Passing `-b` as a command line argument, e.g. `python -b my_pyroot_script.py`. For this to work, `PyConfig.IgnoreCommandLineOptions` must be set to `False` inside the PyROOT script, as discussed [here]({{ '/manual/python/#configuration-options' | relative_url }}).
+
+2. Calling `gROOT.SetBatch` in the PyROOT script, right after importing `ROOT`:
+
+```python
+import ROOT
+ROOT.gROOT.SetBatch(True)
+```
+
+### Low-level manipulation of objects
+
+When instantiating a C++ class from Python via PyROOT, both a C++ object and its Python proxy object are created. Such Python object forwards any access to its internal C++ object, thus acting as a proxy. PyROOT offers some functions to inspect or manipulate Python proxies and their C++ counterparts, based on functionality provided by [cppyy](https://cppyy.readthedocs.io/en/latest/lowlevel.html). All these functions can be accessed as `ROOT.NameOfFunction` and they are listed next:
+
+- `AddressOf(obj)`: when applied to a Proxy object `obj`, it returns an indexable buffer of length 1, whose only element is the address of the C++ object proxied by `obj`. The address of the buffer is the same as the address of the address of the C++ object. This function is kept for backwards compatibility with old PyROOT versions.
+
+- `addressof(obj, field, byref)`: similarly to `AddressOf`, it can be used to obtain the address of an internal C++ object from its Python proxy. However, `addressof` returns that address as an integer, not as an indexable buffer. Furthermore, `addressof` accepts two more parameters: `field` and `byref`. `field` can be used to specify the name of a field in a struct, in order to get the address of that field. If `byref` is set to true, `addressof` returns the address of the address of the C++ object. Equivalent to [cppyy's `addressof`](https://cppyy.readthedocs.io/en/latest/lowlevel.html#capsules). Example of usage:
```

Comment:
There's something missing between here: ... the address of the C++ object. Equivalent to  ...

Reply:
I mean to make a valid sentence out of it

Reply:
Example of usage -> Example usage

Reply:
Ok I will add "It is equivalent to..." and ok for the example usage.